application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java

```diff
@@ -176,22 +178,40 @@ RestAction<Message> constructChatGptAttempt(ThreadChannel threadChannel,
             return useChatGptFallbackMessage(threadChannel);
         }
 
+        List<String> ids = new CopyOnWriteArrayList<>();
         RestAction<Message> message =
                 mentionGuildSlashCommand(threadChannel.getGuild(), ChatGptCommand.COMMAND_NAME)
                     .map("""
                             Here is an AI assisted attempt to answer your question 🤖. Maybe it helps! \
                             In any case, a human is on the way 👍. To continue talking to the AI, you can use \
                             %s.
                             """::formatted)
-                    .flatMap(threadChannel::sendMessage);
+                    .flatMap(threadChannel::sendMessage)
+                    .onSuccess(m -> ids.add(m.getId()));
+        String[] answers = chatGPTAnswer.get();
+
+        for (int i = 0; i < answers.length; i++) {
+            MessageCreateAction messageCreateAction = threadChannel.sendMessage(answers[i]);
+
+            if (i == answers.length - 1) {
+                message = message.flatMap(ignored -> messageCreateAction
```

Comment:
more idiomatic for `ignored` is usually `_` (or if that doesnt work), `any`

Reply:
true forgot to change it after testing if it even works